lib/datasets.go

```diff
@@ -65,24 +69,63 @@ func (r *DatasetRequests) List(p *ListParams, res *[]repo.DatasetRef) error {
 	return err
 }
 
-// Get retrieves a dataset head (commit, structure, meta, etc) for a given reference, either
-// from the local repo or by asking peers for it. The res parameter will be populated upon success.
-func (r *DatasetRequests) Get(ref *repo.DatasetRef, res *repo.DatasetRef) error {
+// Get retrieves datasets and components for a given reference. If p.Ref is provided, it is
+// used to load the dataset, otherwise p.PathString is parsed to create a reference. The
+// dataset will be loaded from the local repo if available, or by asking peers for it.
+// Using p.Selector will control what components are returned in res.Bytes. The default,
+// a blank selector, will also fill the entire dataset at res.Data.
```

Comment:
does that dataset include the body?

Reply:
Expanded the comment to clarify that if p.Selector == "body", then the body is the in res.Bytes.